### PR TITLE
Fixes 297: Missing access token causes lengthy API calls.

### PR DESCRIPTION
**Pull request by grasmash**

### CODE REVIEW

**Comment by grasmash** [comment]
Looks like this broke Windows:
https://github.com/typhonius/acquia-php-sdk-v2/commit/a301ebd5ef18d7cd2f314dfca1fda9e660bb6155

With:
```
Error: Call to undefined function AcquiaCloudApi\Connector\posix_getpwuid()
```

**Comment by grasmash** [comment]
Blocked by https://github.com/typhonius/acquia-php-sdk-v2/issues/101

**Comment by typhonius** [comment]
I've just committed a change that should fix this if you want to kick off CI again.

**Comment by grasmash** [comment]
Roger that.

File: .travis.yml [hunk 1]
```diff
@@ -54,6 +55,7 @@ jobs:
 before_install:
   # Tests for the SSH key commands rely on ssh-agent running.
   - eval "$(ssh-agent -s)"
+  - composer self-update --preview
```
Comment (grasmash) [inline .travis.yml:58]: This updates to Composer 2.

**Comment by grasmash** [comment]
@danepowell you good with this?

File: composer.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "symfony/validator": "^5.1",
     "symfony/yaml": "^5.1@dev",
     "typhonius/acquia-logstream": "dev-master",
-    "typhonius/acquia-php-sdk-v2": "^2.0.14",
+    "typhonius/acquia-php-sdk-v2": "dev-master as 2.0.14",
```
Comment (danepowell) [inline composer.json:39]: Looks like 2.0.15 is out now, I'd prefer using `^2.0.15` since aliases lead to weird dependency conflicts.

**Review by danepowell — APPROVED** [review]
Would prefer a stable dependency on the sdk but not a huge deal. Otherwise looks great.

**Pull request merged**